components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/resources/org.wso2.carbon.identity.configuration.management.yaml

```diff
@@ -365,18 +365,18 @@ paths:
       x-wso2-curl: |
         curl -k -X GET 'https://localhost:9443/api/identity/config-mgt/v1.0/resource/e-mail/smtp' -H "accept: application/json" -H 'Authorization: Basic YWRtaW46YWRtaW4='
       x-wso2-response: |
-        {"resourceId":"7060b0c6-000b-47ed-86ef-396ff5cb8203","tenantDomain":"carbon.super","resourceName":"smtp","resourceType":"e-mail","lastModified": "2019-01-24T10:15:51Z","created": "2019-01-24T10:15:51Z","attributes":[{"id":"463e5d5b-6362-43f8-96a0-e19343896841","key":"from","value":"3abc.com"},{"id":"a83f1575-b996-4889-9363-16ce295a19be","key":"to","value":"3wso2.com"}],"files":[]}
+        {"resourceId":"6e45c661-7671-4ee9-805c-8d3d1df46cbc","tenantDomain":"carbon.super","resourceName":"smtp","resourceType":"e-mail","lastModified":"2019-02-07T09:31:21.564Z","created":"2019-02-07T09:30:12.963Z","attributes":[{"key":"from","value":"admin@wso2.com"},{"key":"to","value":"abc.com"}],"files":[]}
       summary: |
-        Retrieve the resource.
+        Retrieve resource
       parameters:
         - name: resource-name
           in: path
-          description:  This represents the name of the resource to be retrieved.
+          description:  This represents the name of the resource that is to be retrieved.
           required: true
           type: 'string'
         - name: resource-type
           in: path
-          description:  This represents the type of the resource to be added and can either be the name or id.
+          description:  This represents the type of resource that is to be added and can either be the resource-type name or ID.
```

Comment:
This value can only be the name, not the id. Shall we change it accordingly?

Reply:
Fixed with [c2af0a2](https://github.com/wso2/carbon-identity-framework/commit/c2af0a29f9313baf5105a4898a9e26baceae00fa)